wordle-web/app.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <div>
+    <NuxtPage />
+  </div>
```

Comment:
```suggestion
    <NuxtPage />
```

This `<div>` tag isn't doing anything so it should be removed

Reply:
Whenever I create a nuxt app, it defaults to using the <div> root component. Is there not a reason for it?